### PR TITLE
fix: incorrect authority for keepers

### PR DESCRIPTION
**Pull request by j75689**
### Description

1. fix incorrect authority for `VirtualgroupKeeper` and `OracleKeeper`
2. fix `GvgStakingPerBytes` compare function in  VirtualgroupKeeper.MsgServer
3. add e2e test UpdateParams for each module
4. add virtual group swagger

### Rationale

`authority` should be `Gov` module account.

### Example

```bash
make e2e_start_localchain && make e2e_test
```

### Changes

Notable changes: 
* e2e
* keeper's authority

### CODE REVIEW

**Review by Fynnss — APPROVED** [review]
LGTM

File: e2e/tests/virtualgroup_test.go [hunk 1]
```diff
@@ -543,3 +550,80 @@ func (s *VirtualGroupTestSuite) TestSPExit_CreateAndDeleteBucket() {
...
+			case v1.StatusPassed:
+				s.T().Logf("proposal passed")
+				return
+			case v1.StatusFailed:
+				s.T().Errorf("proposal failed, reason %s", queryProposalResp.Proposal.FailedReason)
+				return
+			}
+		}
+		time.Sleep(1 * time.Second)
+	}
```
Comment (Fynnss) [inline e2e/tests/virtualgroup_test.go:628]: Is it necessary to verify that the parameters have really changed after updated?

**Review by forcodedancing — APPROVED** [review]

File: e2e/tests/virtualgroup_test.go (hunk 1 above)
Comment (guagualvcha) [reply e2e/tests/virtualgroup_test.go:628]: Agree.

Comment (j75689) [reply e2e/tests/virtualgroup_test.go:633]: updated

**Review by guagualvcha — APPROVED** [review]

**Pull request merged**